samples/todolist-aspnetcore/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+## Purpose
```

Comment:
We probably don't need any of the `/.github` files as they are meant for if this was in it's own repository.